src/components/check-your-phone/check-your-phone-controller.ts

```diff
@@ -140,31 +140,32 @@ async function handleAddMfaMethod(
   service: CheckYourPhoneServiceInterface,
   trace: string
 ): Promise<boolean> {
-  const smsMFAMethod = req.session.mfaMethods.find(
+  const defaulMfaMethod = req.session.mfaMethods.find(
     (mfa) => mfa.priorityIdentifier === "DEFAULT"
```

Comment:
```suggestion
    (mfa) => mfa.priorityIdentifier === "DEFAULT" && mfa.method.mfaMethodType === "SMS"
```

If I'm undestanding the logic correctly here, this would mean we could get rid of the if statement on line 154?

Reply:
annoyingly get type error's with this change, without the if statement, it believe's the mfa method could still be auth app and doesn't allow for the use of phoneNumber